### PR TITLE
Add "Outputs" Section Docs

### PR DESCRIPTION
**Pull request by erikerikson**
## What did you implement:
TL;DR: Enrich docs - add "Outputs" section

Adds the "Outputs" section of the CFT to the "everything" example `serverless.yml` documentation.  That example appears to propose to show all the things that can be added to serverless.yml but does not show the valid addition of an "Outputs" section, so this PR adds one.

## How did you implement it:

Edit to docs.

## How can we verify it:

Ask yourself, do the changes make sense and is it a valid addition to a serverless.yml?

## Todos:

- [x] Write tests
- [x] Write documentation
- [x] Fix linting errors
- [x] Make sure code coverage hasn't dropped
- [x] Provide verification config/commands/resources
- [x] Enable ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) for this PR
- [x] Change ready for review message below

***Is this ready for review?:*** YES

### CODE REVIEW

**Comment by erikerikson** [comment]
Except for the part where apparently custom Outputs are stripped from the serverless.yml?
Did this just become a feature request?

**Comment by pmuens** [comment]
Great addition! Thanks @erikerikson !

**Comment by nikgraf** [comment]
Oh yeah! Really great addition @erikerikson
